scripts/testnet/sovereignBridge/prerequisites.sh

```diff
@@ -1,31 +1,33 @@
 #!/bin/bash
 
-runBridgePrerequisites() {
-    installSoftwareDependencies
+echo "Installing software prerequisites..."
+sudo apt update
+sudo apt install -y python3-pip pipx screen ca-certificates curl wget
+pip install multiversx-sdk
+pipx install multiversx-sdk-cli --force
+pipx ensurepath
 
-    downloadCrossChainContracts
-}
+source config/configs.cfg
+source config/contracts.snippets.sh
+downloadCrossChainContracts
```

Comment:
These 3 extra functions should not be done by default.
Please put them inside an `if SOVEREIGN_CHAIN`

Reply:
This bridge software prerequisites and we need this by default.